x-pack/plugins/ml/public/application/components/page_header/page_header.tsx

```diff
@@ -14,7 +14,7 @@ import { MlPageControlsContext } from '../ml_page/ml_page';
 /**
  * Component for setting the page header content.
  */
-export const MlPageHeader: FC = ({ children }) => {
+export const MlPageHeader: FC<{ children?: React.ReactNode }> = ({ children }) => {
```

Comment:
There appears to still be some places in ML code where `PropsWithChildren` could be used.

Reply:
Thank you @jgowdyelastic 🙇

Reply:
There are still quite a lot of places in the ML code where `PropsWithChildren` can be used rather than `{ children?: React.ReactNode }`.
But to not hold up this PR I'll approve and can fix them in a follow up.